modules/retail/pages/retail-install-setup.adoc

```diff
@@ -1,42 +1,69 @@
 [[retail-install-setup]]
 = Setting up
 
-When you have installed the [guimenu]``SUSE Manager for Retail Server`` and the [guimenu]``SUSE Manager Retail Branch Server``, you need to set up your environment by following these steps:
+This section requires an installed and configured {susemgr} Server, one or more {smr} branch servers, and one or more {susemgr} build host.
 
-. Install the build host and register it to {susemgr}
-. Configure the build host
-. Create the required system group
+Next steps in configuring {smr} environment are:
 
-These are the branch server setup steps:
+. Prepare POS images
+. Configure {smr} branch server services
+. Synchronize POS images to {smr} branch servers
 
-// already done:
-// . Install the branch server and register it to {susemgr}
-. Assign and configure branch server formulas
-. Synchronize images to the branch servers
+These steps are usually performed multiple times in lifetime of {smr} and are loosely related.
```

Comment:
What is this sentence trying to say? "You'll do this a lot, get used to it?"

Reply:
In the typical setup customer have only one or small number of build hosts and images, so this step (POS Images) is done only few times. But then have tens or hundreds, even thousands of branch servers and these needs to be setup all the time. 

I structured the doc that after suma is configured, build host is configured and image is build and then numerous branches are configured and then images are synchronized.

But for the first branch, you need to have images to synchronize. For the second branch, you do not need new image. You can reuse the existing one so you skip POS Images step.
When you will build new image and you don't have to do any reinitialization of branches, just synchronize them, so you skip branch services configuration step.

There is some relation between images and branches, but it is not that one require the other, but rather that one needs other for working system and both parts are changing during the product lifetime.

Now how do we write this into the documentation in understandable way?

Reply:
For starters, it's going to need more than three sentences, I think!

Reply:
I can't check out your branch to edit directly, but I would do something like this (from line 4):

`
To set up an {smr} environment, you will need to have already installed and configured {susemgr} Server, have one or more {smr} branch server, and one or more {susemgr} build host.

This section covers how to configure your {smr} environment, including:
. Prepare POS images
￼. Configure services on the branch server
. Synchronize POS images to the branch servers

The very first time you set up an {smr} environment, you will need to perform all three steps.
You will need to revisit some of these steps later on as you are working with {smr}.

For example, the first time you configure the branch server, you will need to have images prepared for synchronization.
If you are configuring more than one branch server, you will only need to synchronize images for the first branch server.
Subsequent branch servers can use the same image.

If you have an existing environment, and need to build new images, you do not need to re-initialize the branches.
You will need to synchronize the images, and can skip setting up the services on the branch server.
`